### DISCUSSION TITLE
Faction Information Suggestions

### DISCUSSION THREAD

**Discussion by Kalinter** [globemallow-mw/ossuaries-of-the-ghostfence#30] [Ideas]
Faction Info

Part 1: Game statistics

Favored Attributes:
Speed - expected to be able to move quickly through the catacombs
Willpower - need a strong will to brave the catacombs

Favored skills:
Athletics - see above
Medium Armor or Light Armor - we could set their uniform (sacred necromancer armor) as either but it certainly doesn’t look like heavy armor 
Blunt - quickly bash skellingtons
Conjuration - summon undead and turn undead
Illusion - light spells and night eye to brave the darkness of the catacombs 
Maybe Mysticism - detect spells - Or maybe a secondary weapon type- short blade?

Faction Ranks and duties expected of each rank
Pilgrim - complete the pilgrimage
Initiate - initiated into the order
Scout - scouts sent into the catacombs and return quickly
Runner - runners expected to make it quickly through the catacombs between the ossuaries
Sergeants - sergeants do task for the order on the surface away from Red Mountain
Guardian - guardians serve for long periods of time being stationed in the catacombs
Defender - Defenders stationed at an ossuary or the Hall of Ghosts as Ghostgate permanently- seen as a cushiony rank in the order 
Commandant - commandants command a group of guardians in the catacombs
Prefect - One prefect at each ossuary, player can become the new prefect of Vekh, as the old one will die during our questline
Commander - Leader of the whole group, Player probably won’t become commander unless we want them to retire or die during the questline

Part 2: Advancement

The above info is mostly just flavor text that we could implement in some way. Faction advancement can probably be just handled by the story as we need it, and not done in the traditional Morrowind way.

Part 3: Special Quest reward ideas

First Reward: when becoming a full member of the order the player receives a full set of the Order’s armor (Sacred Necromancer Armor)
Second Reward: you get a weapon of the order - bone weapon?
Third Reward: Strong turn undead power 
Fourth Reward: power to summon a small army of undead
Final Reward: Player can takeover the Prefect quarters at the ossuary of Vekh as an isolated, but unique player home. Player also gets a spell that allows them to pass through ghostfences (Both THE Ghostfence and Ghostfences in the Catacombs. )

**Comment by dev-chenxing** [comment]
New Class for the members of the faction: 

Class Id: GG_Crusader
Class Name: Spirit Knight

Specialization: Magic

Attributes: Speed, Willpower

Major skills:
Conjuration, Light Armor, Blunt Weapon, Athletics, Illusion

Minor skills:
Medium Armor, Mysticism, Destruction, Restoration, Enchant

**Comment by dev-chenxing** [comment]
Faction ID: GG_Order_of_Ghosts
Faction Name: Order of Ghosts

Question: Are we going to set attribute and skill requirement for faction rank advancement or not?

**Reply by Kalinter** [reply]
I would vote to just have faction advancement handles solely by the storyline, So I wouldn't really worry about it, IMO.
